src/yetibot/commands/twitter.clj

```diff
@@ -63,6 +63,13 @@
       (format "Stopped tracking %s" topic))
     (format "You're not tracking %s" topic)))
 
+(defn show
+  "twitter show <screen-name> # show top 10 tweets from user <scree-name>"
+  [{[_ screen-name] :match}]
+  (let [tweets (:body (model/user-timeline screen-name 10))]
+    (into [] (map #(model/format-tweet-text %) tweets ))))
```

Comment:
1. You can simplify `(map #(model/format-tweet-text %) tweets)` to be `(map model/format-tweet-text tweets)`
1. Do you need the `into []` here? The output of `map` will already be a collection so you should be able to omit